sympy/utilities/lambdify.py

```diff
@@ -202,75 +208,253 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     ``x`` and SymPy functions ``sin`` and ``cos``, then converted it to an
     equivalent NumPy function ``f``, and called it on a NumPy array ``a``.
 
-    .. warning::
-       This function uses ``exec``, and thus shouldn't be used on unsanitized
-       input.
+    Parameters
+    ==========
 
-    Arguments
-    =========
+    args : List[Symbol]
+        A variable or a list of variables in the expression.
```

Comment:
A variable or a list of variables whose nesting represents the nesting of the arguments that will be passed to the function.

>>> add1 = lambdify(x, x + 1)  # expect 1 arg
>>> add1(1)
2
>>> add = lambdify([x, y], x + y)  # expect 2 args
>>> add(1, 1)
2
>>> is_pythag = lambdify([[a, b, c]], Eq(c**2 ,a**2 + b**2))  # expect 1 arg of length 3
>>> t = (3, 4, 5)
>>> is_pythag(t)
True
>>> f = lambdify([x, (y, z)], x + y + z)  # expect 2 args: scalar and tuple of length 2
>>> f(1, (2, 3))
6
```